src/WebJobs.Script.Grpc/azure-functions-language-worker-protobuf/src/proto/FunctionRpc.proto

```diff
@@ -78,7 +78,7 @@ message StreamingMessage {
 
```

Comment:
Can you raise a separate PR for bumping the proto version?

https://github.com/Azure/azure-functions-language-worker-protobuf#pulling-updates

Reply:
I tried this several times. Somehow, the pulling updates is not working for me (though I have done this few months back successfully). Let me know if you have done this recently. We can chat offline and get this done.